src/System.Net.WebSockets.WebSocketProtocol/ref/System.Net.WebSockets.WebSocketProtocol.csproj

```diff
@@ -9,4 +9,10 @@
   <ItemGroup>
```

Comment:
@anurse (and @ericstj), is this library still needed?  It was only being built as a stop-gap for ASP.NET, but ASP.NET is no longer using it, right?  Can we just stop building it?

Reply:
Correct, in 3.0 we're using WebSocket.CreateFromStream and don't need this.

Reply:
So if you wanted to stop building the library and package it would meant that you'd need to go to a servicing branch if you ever wanted to update it.  It'd also be take-back from Preview7 if anyone was depending on changes we made this release.

Reply:
> It'd also be take-back from Preview7 if anyone was depending on changes we made this release.

This PR is for post-3.0.

Reply:
@benaadams, I'd feel more comfortable if we removed the lib as a dedicated PR rather than as part of this one.  That'd also make it easier to revert if we need to.

Reply:
Undid